runtime/compiler/trj9/optimizer/MonitorElimination.cpp

```diff
@@ -1464,6 +1464,26 @@ void TR::MonitorElimination::transformMonitorsIntoTMRegions()
       TR::CFG *cfg = comp()->getFlowGraph();
       cfg->setStructure(NULL);        // voiding out structure
 
+      if( monitor->getTreeTopNode()->getOpCodeValue() == TR::NULLCHK)
```

Comment:
Parenthesis formatting fixup needed here and at the below `TR_ASSERT` so we're consistent with the rest of the codebase.